js-extensions/src/ResourceLoadTracker.js

```diff
@@ -69,12 +69,14 @@ export default class ResourceLoadTracker {
      *
      * @param extensionPointName The extension point name.
      */
-    onMount(extensionPointName) {
+    onMount(extensionPointName, onload) {
         const pointCSS = this.pointCSSs[extensionPointName];
         if (pointCSS) {
             for (var i = 0; i < pointCSS.length; i++) {
-                this._requireCSS(pointCSS[i]);
+                this._requireCSS(pointCSS[i], onload);
             }
+        } else {
+            onload();
```

Comment:
you never test that `onload !== null || onload !== undefiended` and you do not set a default. That can lead to NPE `null()`

Reply:
again, i want to make sure it's passed in

Reply:
IMO If a function makes no sense without a specific argument, we *want* it to error out, not just silently do nothing. If that means we need a bullshit handler in tests, that's how tests go.

Reply:
I agree with @sophistifunk if `!onload` then we should throw an controlled error and not `null is not a function` stacktrace that does not help 

something like the following makes more sense IMHO

```
if (!onLoad) throw new Error('onLoad function is not passed to the component')
```